BeeSwift/RemoveHKMetricViewController.swift

```diff
@@ -53,14 +54,20 @@ class RemoveHKMetricViewController: UIViewController {
     @objc func removeButtonPressed() {
         guard self.goal != nil else { return }
         self.goal?.autodata = ""
-        var params : [String : [String : String]] = [:]
-        params = ["ii_params" : ["name" : "", "metric" : ""]]
+        var params: [String: [String: String]] = [:]
```

Comment:
since we touched these lines (formatting), just as well merge the two into `let params`